content/testing.md

```diff
@@ -355,24 +355,27 @@ In the [unit test above](#simple-unit-test) we saw a very limited example of how
   - Alternatively, you can also use tools like [Sinon](http://sinonjs.org) to stub things directly, as we'll see for example in our [simple integration test](#simple-integration-test).
 
   - The [`hwillson:stub-collections`](https://atmospherejs.com/hwillson/stub-collections) package we mentioned [above](#mocking-the-database).
+  
+There's a lot of scope for better isolation and testing utilities.
 
-  - (Using another package from the example app) to isolate a publication, the `publication-collector` package:
-
-    ```js
-    describe('lists.public', function () {
-      it('sends all public lists', function (done) {
-        // Allows us to look at the output of a publication without
-        // needing a client connection
-        const collector = new PublicationCollector();
-        collector.collect('lists.public', (collections) => {
-          chai.assert.equal(collections.Lists.length, 3);
-          done();
-        });
-      });
-    });
-    ```
+<h4 id="testing-publications">Testing publications</h4>
 
-There's a lot of scope for better isolation and testing utilities (the two packages from the example app above could be improved greatly!). We encourage the community to take the lead on these.
+Using the [`publication-collector` package](https://atmospherejs.com/johanbrook/publication-collector), you're able to test individual publication's output without needing to create a traditional subscription:
+
+```js
+describe('lists.public', function () {
+  it('sends all public lists', function (done) {
+    // Allows us to look at the output of a publication without
+    // needing a client connection
```

Comment:
remove this comment? already said above. maybe replace with `// Set the userId that will be provided to the publish function`